app/views/faq.scala.html

```diff
@@ -283,42 +293,78 @@ <h3 id="metadata-captured" class="govuk-heading-s">What metadata will be capture
 
             <h3 class="govuk-heading-s">Why are you asking me to add descriptive metadata?</h3>
             <p class="govuk-body">
-                During the upload process, TDR runs DROID to capture basic file-level metadata. However, if you have additional information about your records, you should add this. We strongly recommend that you supply this information to us. It can include anything from descriptions to additional dates and any additional data that will provide context to the record and make it easier to find in the future. To do this during private beta, you will need to download DROID and create a metadata CSV which contains the additional metadata. For further assistance, contact <a href="mailto:@Messages("nationalArchives.email")">
+                During the upload process, TDR captures basic file-level metadata. If you have additional information about your records, we recommend that you add this. It can include anything from descriptions to dates and any additional data that will provide context to the record and make it easier to find in the future. For further assistance, contact <a href="mailto:@Messages("nationalArchives.email")">
                 @Messages("nationalArchives.email")</a>.
             </p>
 
+            <h3 class="govuk-heading-s">What do I do if the title or description of my record is sensitive to the public?</h3>
+            <p class="govuk-body">
+                If the title or description of your record is sensitive to the public, you can provide an alternative when adding closure metadata. The alternative will be visible on the public catalogue. You may wish to do one of the following:
+            </p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>Redact - remove the sensitive words. For example: 'Correspondence with [name withheld]'.</li>
+                <li>Sanitise - replace the sensitive word. For example: 'UK military exercises in xx' (where 'military' has replaced 'Special Forces').</li>
```

Comment:
The single quotes on this line don't exactly match the single quotes in the document, although I think this way looks better.

Reply:
Bit of editorial licence, as the quotes in the doc didn't make sense as they were.